easybuild/framework/easyconfig/types.py

```diff
@@ -400,6 +400,12 @@ def to_dependencies(dep_list):
     """
     return [to_dependency(dep) for dep in dep_list]
 
+def to_checksums(checksums):
+    if not isinstance(checksums[0], list):
```

Comment:
checksums can be a list of lists, see https://github.com/hpcugent/easybuild-framework/blob/master/test/framework/easyconfigs/toy-0.0.eb#L10